Scraper/webScraper.py

```diff
@@ -0,0 +1,22 @@
+from bs4 import BeautifulSoup
+import requests
+
+# Save site url in a variable
+URL = "http://smashboards.com/threads/captain-falcon-hitboxes-and-frame-data.284165/"
+
+# Open the url and save the html data into bs4
+def request_page(link):
+    r = requests.get(link)
+    data = r.text
+    webpage = BeautifulSoup(data, 'html.parser')
+    return webpage
+
+def parse_info(webpage):
```

Comment:
Unless you foresee using the bs4 webpage for other operations, I would just have **request_page** return the text from the page, and rename it to something like **parse_page**. Just whatever name you feel would best convey the operation of the function.